packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -234,9 +243,7 @@ const ProjectAccess: FC<ProjectAccessProps> = ({
                                 projectUuid={projectUuid}
                                 canManageProjectAccess={canManageProjectAccess}
                                 user={orgUser}
-                                inheritedRoles={
-                                    inheritedRoles[orgUser.userUuid]
```

Comment:
this was not checking if userUuid was in `inheritedRoles` - it could result on an error

Reply:
more on this - a race condition could lead to an error